app/content/views/user.py

```diff
@@ -273,14 +273,41 @@ def get_user_detail_strikes(self, request, *args, **kwargs):
         )
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    # @action(detail=False, methods=["get"], url_path="me/events")
+    # def get_user_events(self, request, *args, **kwargs):
+    #     registrations = request.user.registrations.all()
+    #     events = [
+    #         registration.event
+    #         for registration in registrations
+    #         if not registration.event.expired
+    #     ]
+    #     return self.paginate_response(
+    #         data=events, serializer=EventListSerializer, context={"request": request}
+    #     )
+
     @action(detail=False, methods=["get"], url_path="me/events")
     def get_user_events(self, request, *args, **kwargs):
         registrations = request.user.registrations.all()
+
+        # Apply the filter
+        event_has_ended = self.request.query_params.get("expired") == "true"
```

Comment:
Her burde dere bruke get sitt andre mulige parameter, som er default verdi hvis man ikke finner parameteret. 
Legg også til en .lower() for å få verdien fra param i lowercase uansett hva som blir sendt inn.
Slik:

self.request.query_params.get("expired", False).lower() == "true"